google-map.html

```diff
@@ -431,7 +441,7 @@
         return;
       }
       this._mutationObserver = new MutationObserver( this._updateMarkers.bind(this));
-      this._mutationObserver.observe(this, {
+      this._mutationObserver.observe(this.$.selector, {
```

Comment:
check that this works under native shadow dom (?dom=shadow on the url will enable it). This changes the MO to watch for children of `<iron-selector>`, but no the light dom google-map-markers. The children of <iron-selector> is only the `<content>`.